tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -235,6 +235,17 @@ mod generated_constants {
     fn monster_file_extension() {
         assert_eq!("mon", my_game::example::MONSTER_EXTENSION);
     }
+
+    #[test]
+    fn enum_constants() {
```

Comment:
`enum_constants` does not convey the intention we want. I would prefer a name like `enum_constants_are_public`.

Reply:
Changed to `enum_constants_are_public`.